From f7cfa25bff470461da7b0b2d0e2c2cfeb5f072f1 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Thu, 13 Oct 2016 10:18:17 +0800
Subject: [PATCH 1215/5242] MLK-13334 ARM: dts: imx: Add lpddr2 arm2 dts for
 imx6sll

commit  89968c49769c0c974d1c55254d25144346db0288 from
https://source.codeaurora.org/external/imx/linux-imx.git

For i.MX6SLL LPDDR2 and LPDDR3 ARM2 board, they share the same
board design but using different DDR chip. So we can reuse the
LPDDR3 ARM2 board dts on LPDDR2 ARM2 board.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile                |    1 +
 arch/arm/boot/dts/imx6sll-lpddr2-arm2.dts |   10 ++++++++++
 2 files changed, 11 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6sll-lpddr2-arm2.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index bfd7a59..b6dbb87 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -629,6 +629,7 @@ dtb-$(CONFIG_SOC_IMX6ULL) += \
 	imx6ull-9x9-evk.dtb \
 	imx6ull-9x9-evk-btwifi.dtb
 dtb-$(CONFIG_SOC_IMX6SLL) += \
+	imx6sll-lpddr2-arm2.dtb \
 	imx6sll-lpddr3-arm2.dtb \
 	imx6sll-lpddr3-arm2-csi.dtb \
 	imx6sll-lpddr3-arm2-ecspi.dtb \
diff --git a/arch/arm/boot/dts/imx6sll-lpddr2-arm2.dts b/arch/arm/boot/dts/imx6sll-lpddr2-arm2.dts
new file mode 100644
index 0000000..33f230f
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sll-lpddr2-arm2.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sll-lpddr3-arm2.dts"
+
-- 
1.7.9.5

